utils/load_path.mli

```diff
@@ -60,7 +60,16 @@ module Dir : sig
       sub-directories of this directory. *)
 end
 
-val add : Dir.t -> unit
+val[@deprecated] add : Dir.t -> unit
```

Comment:
Is it worth deprecating a function in compiler-libs ? Why not simply remove it ?

Reply:
Given all the Merlin stuff and everything else I didn't have the courage to do this! I don't have strong opinions on the names at all - `append_dir` / `prepend_dir` seemed to make for a clearer API than `add` and `add_at_start`, for example. The compiler doesn't use `Load_path.append_dir` anyway.